pidfile/pidfile_test.go

```diff
@@ -31,7 +32,16 @@ func TestCreate(t *testing.T) {
 	if pid != os.Getpid() {
 		t.Errorf("contents of pidfile does not match pid. content: %d, pid: %d", pid, os.Getpid())
 	}
+}
 
+func TestCreate_mutex(t *testing.T) {
+	dir, err := ioutil.TempDir("", "")
+	if err != nil {
+		t.Fatalf("failed to create tempdir")
+	}
+	defer os.RemoveAll(dir)
+	pidfile := filepath.Join(dir, "pidfile")
+	ioutil.WriteFile(pidfile, []byte(fmt.Sprintf("%d", os.Getppid())), 0644)
```

Comment:
If I understand correctly, `s/os.Getppid()/os.Getpid()/` and `err` should be nil.